renter/renterutil/filesystem_test.go

```diff
@@ -117,9 +118,16 @@ func TestHostErrorSet(t *testing.T) {
 	}
 	// should get a HostErrorSet when we sync
 	err = pf.Sync()
-	hes, ok := errors.Cause(err).(HostErrorSet)
+	for {
+		uerr := errors.Unwrap(err)
+		if uerr == nil {
+			break
+		}
+		err = uerr
+	}
+	hes, ok := err.(HostErrorSet)
```

Comment:
```go
var hes HostErrorSet
if !errors.As(err, &hes) {
    t.Fatal("expected HostSetError, got", err)
````

Reply:
I figured this one was coming, but I glossed over the `As` docs, saw "something something `panic` if something", and tried to maintain the existing implementation as much as possible. Will fix.